### PR TITLE
[collections] extend collectible with new fields

### PR DESCRIPTION
**Pull request by ivan-bozhytksyi**

### CODE REVIEW

File: types/collectibles.go [hunk 1]
```diff
@@ -53,6 +53,14 @@ type (
 		TransferFee       *CollectibleTransferFee `json:"transfer_fee,omitempty"`
 		PreviewImageURL   *CollectibleMedia       `json:"preview_image_url,omitempty"`
 		OriginalSourceURL CollectibleMedia        `json:"original_source_url"`
+		Properties        []CollectibleProperty   `json:"properties"`
+		About             string                  `json:"about"`
```
Comment (vikmeup) [inline types/collectibles.go:57]: what does about contains?

**Review by vikmeup — APPROVED** [review]

File: types/collectibles.go (hunk 1 above)
Comment (ivan-bozhytksyi) [reply types/collectibles.go:57]: It contains contract description

File: types/collectibles.go [hunk 2]
```diff
@@ -53,6 +53,14 @@ type (
...
+	}
+
+	CollectibleProperty struct {
+		Key    string  `json:"key"`
+		Value  string  `json:"value"`
+		Rarity float32 `json:"rarity,omitempty"`
```
Comment (dimkouv) [inline types/collectibles.go:63]: Can Rarity be zero?
If yes then omitempty will exclude it from json, otherwise it's ok.

**Review by dimkouv — APPROVED** [review]

File: types/collectibles.go (hunk 2 above)
Comment (ivan-bozhytksyi) [reply types/collectibles.go:63]: Yes, it can be zero and will be excluded from response

**Pull request merged**